packages/compose/src/hooks/use-drop-zone/index.js

```diff
@@ -219,6 +219,12 @@ export default function useDropZone( {
 			ownerDocument.addEventListener( 'dragenter', maybeDragStart );
 
 			return () => {
+				onDropRef.current = null;
+				onDragStartRef.current = null;
+				onDragEnterRef.current = null;
+				onDragLeaveRef.current = null;
+				onDragEndRef.current = null;
+				onDragOverRef.current = null;
```

Comment:
fun fact! `useRef` already accounts for `undefined` as a possibility, so if we unset these with `onDropRef.current = undefined` then we don't need to change the type signature in line 15.

Reply:
Let's see if we can get away with removing useFreshRef in https://github.com/WordPress/gutenberg/pull/39038/commits/9020dfd8a9c551acd1472c3610f8d58ec24bced3